exporter/collector/googlemanagedprometheus/naming_test.go

```diff
@@ -46,13 +46,25 @@ func TestGetMetricName(t *testing.T) {
 		expectErr bool
 	}{
 		{
-			desc:     "sum without total is unaffected",
+			desc:     "sum without total",
 			baseName: "foo",
 			metric: func(m pmetric.Metric) {
 				m.SetName("foo")
 				sum := m.SetEmptySum()
 				sum.SetIsMonotonic(true)
 			},
+			expected: "foo_total/counter",
+		},
+		{
+			desc:     "sum without total is unaffected when normalization disabled",
+			baseName: "foo",
+			metric: func(m pmetric.Metric) {
+				//nolint:errcheck
```

Comment:
The default for this feature gate changed, I tried to preserve tests with and without the change